common/src/main/java/org/apache/celeborn/common/network/TransportContext.java

```diff
@@ -177,7 +177,8 @@ private TransportChannelHandler createChannelHandler(
         conf.connectionTimeoutMs(),
         closeIdleConnections,
         enableHeartbeat,
-        conf.clientHeartbeatInterval());
+        conf.clientHeartbeatInterval(),
+        this);
```

Comment:
this -> conf.getModuleName() ? May be better if only modulename is used in the log？